votingpool/db_wb_test.go

```diff
@@ -80,3 +81,78 @@ func TestGetMaxUsedIdx(t *testing.T) {
 		t.Fatalf("Wrong max idx; got %d, want %d", maxIdx, Index(3001))
 	}
 }
+
+func TestWithdrawalSerialization(t *testing.T) {
+	tearDown, pool, store := TstCreatePoolAndTxStore(t)
+	defer tearDown()
+
+	roundID := uint32(0)
+	wi := createAndFulfillWithdrawalRequests(t, pool, store, roundID)
+
+	serialized, err := serializeWithdrawal(
+		wi.requests, wi.startAddress, wi.lastSeriesID, wi.changeStart, wi.dustThreshold,
```

Comment:
to match the rest of the codebase, please put the first arguments on the same line as the function name if it's not an excessively long line.